transit_odp/organisation/tests/csv/test_service_codes.py

```diff
@@ -1,82 +1,101 @@
     otc_lic2 = LicenceModelFactory(number=licence_number2)
-    service_code2 = f"{licence_number2}:1"
-    service_code3 = f"{licence_number2}:2"
-    service_code6 = f"{licence_number2}:3"
-    ServiceModelFactory(licence=otc_lic2, registration_number=service_code2)
-    ServiceModelFactory(licence=otc_lic2, registration_number=service_code3)
-    ServiceModelFactory(licence=otc_lic2, registration_number=service_code6)
+    registration_number2 = f"{licence_number2}/1"
+    registration_number3 = f"{licence_number2}/2"
+    registration_number6 = f"{licence_number2}/3"
+    ServiceModelFactory(licence=otc_lic2, registration_number=registration_number2)
+    ServiceModelFactory(licence=otc_lic2, registration_number=registration_number3)
+    ServiceModelFactory(licence=otc_lic2, registration_number=registration_number6)
 
     licence_number3 = "PB000003"
     otc_lic3 = LicenceModelFactory(number=licence_number3)
-    service_code4 = f"{licence_number3}:1"
-    service_code5 = f"{licence_number3}:2"
-    ServiceModelFactory(licence=otc_lic3, registration_number=service_code4)
-    ServiceModelFactory(licence=otc_lic3, registration_number=service_code5)
+    registration_number4 = f"{licence_number3}/1"
+    registration_number5 = f"{licence_number3}/2"
+    ServiceModelFactory(licence=otc_lic3, registration_number=registration_number4)
+    ServiceModelFactory(licence=otc_lic3, registration_number=registration_number5)
 
     org1 = OrganisationFactory()
     BODSLicenceFactory(organisation=org1, number=licence_number1)
     bods_lic = BODSLicenceFactory(organisation=org1, number=licence_number2)
     org2 = OrganisationFactory()
     BODSLicenceFactory(organisation=org2, number=licence_number3)
 
-    # Published but exempted service codes shouldn't be included in response
+    # Published but exempted service codes should be included in response
     ServiceCodeExemptionFactory(
-        licence=bods_lic, registration_code=int(service_code6.split(":")[1])
+        licence=bods_lic, registration_code=int(registration_number6.split("/")[1])
     )
 
     expected_pairs_org1 = [
-        (licence_number1, service_code1),
-        (licence_number2, service_code2),
-        (licence_number2, service_code3),
+        (licence_number1, registration_number1),
+        (licence_number2, registration_number2),
+        (licence_number2, registration_number6),
+        (licence_number2, registration_number3),
     ]
 
     service_codes_csv = ServiceCodesCSV(org1.id)
     queryset = service_codes_csv.get_queryset()
-    assert queryset.count() == len(expected_pairs_org1)
 
+    assert len(queryset) == len(expected_pairs_org1)
     queryset_pairs = [
-        (service.licence.number, service.service_code) for service in queryset
+        (service["licence_number"], service["service_code"]) for service in queryset
     ]
     assert sorted(queryset_pairs) == sorted(expected_pairs_org1)
 
 
 def test_missing_service_code():
```

Comment:
Not sure why this is called `test_missing_service_code`. In fact I'm not really sure what it's testing.

Reply:
Me either!! it's part of the old code! In the previous export there was a column `Service Status`, this test was about `MISSING DATA` case. Today it doesn't test anything at all!! Happy to remove it!